notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-data2vec.ipynb

```diff
@@ -0,0 +1,682 @@
+{
```

Comment:
<p>I'd shorten the time of benchmarking to 15(?) seconds if the results are not very different. The waiting time is now very long...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/785/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>reduced benchmarking time</p>